test/libyul/ewasmTranslationTests/arithmetic_addmod.yul

```diff
@@ -0,0 +1,25 @@
+{
+  sstore(0, addmod(0, 1, 1))
```

Comment:
```
std::exception::what: :2:13: Error: Function expects 8 arguments but got 12.
  sstore(0, addmod(0, 1, 1))
            ^----^
```

Need to fix the definition in the polyfill.

Reply:
done